### PR TITLE
Create generic form expanding function

### PR DESCRIPTION
**Pull request by brandones**
This will be used for medication sections and for contacts in ZL, at least.

### CODE REVIEW

**Comment by brandones** [comment]
CC @MiguelAHPpih , and see the way it's used in the CES form: https://github.com/PIH/openmrs-config-ces/pull/48

File: configuration/pih/scripts/global/expandable.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ *   - Elements with IDs '[id]-[n]' where '[n]' counts the elements (starting from 1)
+ *   - A button with ID 'show-more-[id]'
+ *   - A button with ID 'show-less-[id]'
+ */
+ function setUpExpandableSection(id) {
+  
+  const elements = jq("[id^=" + id + "]").filter(
+    function() { return this.id.match(new RegExp(id + "-\\d+$")) }
+  );
+  console.log(elements);
```
Comment (mogoodrich) [inline configuration/pih/scripts/global/expandable.js:12]: stray debugging line you probably want to delete?

File: configuration/pih/scripts/global/expandable.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  function init() {
+    while (hasValue(elements[numToShow - 1]) && numToShow < elements.length) {
+      numToShow += 1;
+    }
+    update();
+  };
+
+  function update() {
+    console.log('updating for', numToShow);
```
Comment (mogoodrich) [inline configuration/pih/scripts/global/expandable.js:38]: another log statement?

File: configuration/pih/scripts/global/expandable.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+/**
+ * Requires, given an id '[id]':
+ *   - Elements with IDs '[id]-[n]' where '[n]' counts the elements (starting from 1)
+ *   - A button with ID 'show-more-[id]'
+ *   - A button with ID 'show-less-[id]'
+ */
```
Comment (mogoodrich) [inline configuration/pih/scripts/global/expandable.js:6]: I guess it risks become stale quickly, but perhaps add a comment here that links to an example of this in use?

**Review by mogoodrich — APPROVED** [review]
Sweet, thanks @brandones for creating this, and for starting to build a library of reusable JavaScript for our forms in general... just made a few comments...

**Comment by brandones** [comment]
Good catches and good idea, @mogoodrich . Updated.

File: configuration/pih/scripts/global/expandable.js (hunk 3 above)
Comment (brandones) [reply configuration/pih/scripts/global/expandable.js:19]: I added an example to the comment instead (I'd actually misread your comment as suggesting such the first time I read it). Does that seem good?

Comment (mogoodrich) [reply configuration/pih/scripts/global/expandable.js:19]: Even better.

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @brandones !

**Pull request merged**